pyproject.toml

```diff
@@ -45,11 +45,13 @@ path = "dagshub_annotation_converter/__init__.py"
 dependencies = [
   "coverage[toml]>=6.5",
   "pytest",
+  "devtools",
   "pandas-stubs",
   "types-pillow",
   "types-pyyaml",
   "types-lxml",
   "lxml-stubs",
+  "dagshub",
```

Comment:
This is the dev env dependencies.
The regular dependencies are higher (line 27).

This can probably be moved to the client, but I think I'd want to make it be built/defined with hatch there also there then, I think it's much more logical than the `setup.py` mess.